mergify_engine/web/api/queues.py

```diff
@@ -315,6 +323,29 @@ async def repository_queues(
     return queues
 
 
+@router.get(
+    "/repos/{owner}/{repository}/queues/names",  # noqa: FS003
```

Comment:
What about something like `/config` that returns the whole configuration of the queues, not just the names?

Reply:
If there is a need indeed it would be great ! Tho for the new dashboard UI design we only need the name, we could also integrate an info-bubble or something that shows the config for each queues on their respective tables section ?

Reply:
what would be the use case for a user to use this endpoint ?

Reply:
Yes, I think it'd be valuable to know that queue has spec check enabled for example, and show it in the dashboard.
And having one single endpoint and one single call to do is going to be more efficient.